### PR TITLE
Getting closer to 1.0.0

### PR DESCRIPTION
**Pull request by fhunleth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,27 +110,8 @@ run:
     make busybox-menuconfig
 
 Just like the other configuration menus, when you exit menuconfig, the options
-will only be stored in your build directory. To make them permanent, run `make busybox-savedefconfig`
-and save the `.config` (see `build/busybox-*/.config`) to your configuration
-directory. You will need to run `make menuconfig` to update the location of
+will only be stored in your build directory. To make them permanent, save the
+`.config` (see `build/busybox-*/.config`) to your configuration directory. You
+will need to run `make menuconfig` to update the location of
```
[the remark this answers is not in the document]
Comment (fhunleth) [reply README.md]: `make busybox-update-config` only works if you have a `busybox.config` in your system already. If you're using the one from `nerves_system_br`, this doesn't work.

[the remark this answers is not in the document]
Comment (mobileoverlord) [reply README.md]: ah, makes sense, good to know

**Comment by fhunleth** [comment]
Updates to all comments pushed.

**Comment by mobileoverlord** [comment]
looks good to me 
:shipit:

**Comment by mobileoverlord** [comment]
actually hold off a sec, I have a commit to add for updating the docker image

**Comment by mobileoverlord** [comment]
all good, feel free to re arrange / squash my commit if you like. :)

**Pull request merged**